nemo_text_processing/inverse_text_normalization/es/taggers/time.py

```diff
@@ -82,32 +88,59 @@ def __init__(self):
             graph_teen,
             (graph_ties + pynutil.insert("0")),
             (graph_ties + pynutil.delete(" y ") + graph_digit),
-        )
+        ).optimize()
 
         # note that graph_hour will start from 2 hours
         # "1 o'clock" will be treated differently because it
         # is singular
         digits_2_to_23 = [str(digits) for digits in range(2, 24)]
         digits_1_to_59 = [str(digits) for digits in range(1, 60)]
 
-        graph_1oclock = pynini.cross("la una", "la 1")
-        graph_hour = pynini.cross("las ", "las ") + graph_1_to_100 @ pynini.union(*digits_2_to_23)
+        oneoclock = pynini.accep("la una")
```

Comment:
Use underscore `one_o_clock` This is going to cause variable errors if you spell this way.

Reply:
why would it cause variable errors?

Reply:
changed

Reply:
Murphy's law, someone can speed read and see `onoclock` `oneclock`. Readibility save lines.

Reply:
true words are true